src/test/features/controls-spec.ts

```diff
@@ -165,6 +165,29 @@ suite('ModelViewerElementBase with ControlsMixin', () => {
         expect(element.getFieldOfView()).to.be.equal(nextFov);
       });
 
+      test('causes camera-change event to fire', async () => {
+        const cameraChangeDispatches = waitForEvent(element, 'camera-change');
+        const cameraOrbit = element.getCameraOrbit();
+        element.cameraOrbit = `${cameraOrbit.theta + 1}rad ${
+            cameraOrbit.phi}rad ${cameraOrbit.radius}m`;
+
+        await cameraChangeDispatches;
+      });
+
+      test('sets an appropriate event source', async () => {
+        const cameraChangeDispatches =
+            waitForEvent<CustomEvent<CameraChangeDetails>>(
+                element, 'camera-change');
+
+        const cameraOrbit = element.getCameraOrbit();
+        element.cameraOrbit = `${cameraOrbit.theta + 1}rad ${
+            cameraOrbit.phi}rad ${cameraOrbit.radius}m`;
+
+        const event = await cameraChangeDispatches;
```

Comment:
I'm still a little dubious on the value of all this duplicated code, when if the first test fails, it'll just fail both of these anyway.

Reply:
If the code that assigns the camera change source breaks in some way, one test will pass and the other will not.

Reply:
True; is this related to how we seem to get zero test logging feedback in Safari, as compared to Chrome? Because I feel like with the output Chrome gives, that would be easy to disambiguate even if they were only one test.

Reply:
It's more about not conflating behavioral tests. It is advantageous for tests to be based around discrete behaviors or features.

For example, if someday we decided to remove the `event.detail.source` property, we might come along here and delete this test at the same time. If it was the only test that covered the `camera-change` event dispatching in the first place, we might accidentally remove coverage for something because the test was pulling double duty.